modules/reference/pages/properties/topic-properties.adoc

```diff
@@ -353,7 +354,7 @@ When `message.timestamp.type` is set, it overrides the cluster property xref:./c
 
 The maximum size of a message or batch of a topic. If a compression type is enabled, `max.message.bytes` sets the maximum size of the compressed message or batch.
 
-If `max.message.bytes` is set to a positive value, it overrides the cluster property xref:./tunable-properties.adoc#kafka_batch_max_bytes[`kafka_batch_max_bytes`] for the topic.
+If `max.message.bytes` is set to a positive value, it overrides the cluster property xref:reference:properties/cluster-properties.adoc#kafka_batch_max_bytes[`kafka_batch_max_bytes`] for the topic.
```

Comment:
@Deflaimun: Can you please check the link for `default_topic_replication` in this section: https://deploy-preview-334--redpanda-docs-preview.netlify.app/current/reference/properties/topic-properties/#replication-factor

Reply:
Also, please check [Segment size for compacted segments](https://deploy-preview-334--redpanda-docs-preview.netlify.app/current/manage/cluster-maintenance/disk-utilization/#segment-size-for-compacted-segments) in this section: https://deploy-preview-334--redpanda-docs-preview.netlify.app/current/reference/properties/topic-properties/#cleanuppolicy

Reply:
done!